docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-107.adoc

```diff
@@ -32,7 +32,7 @@ An OpenAI API Key is a secure and unique identifier used to authenticate request
 
 *OpenAI*
 
-To address the exposure of an OpenAI API Key, follow these remediation steps to ensure that your access remains secure. OpenAI provides a mechanism to manage API keys directly through its platform:
+If your OpenAI API key is exposed, immediately regenerate a new key through your OpenAI account and replace the compromised key in all your applications and services:
```

Comment:
If your OpenAI API key is exposed, take immediate action to prevent further use by regenerating a new key through your OpenAI account. Replace the compromised key in all your applications and services with the newly generated one. This will ensure continued functionality while mitigating the security risks associated with the exposed key.